features/nanostack/nanostack-interface/NanostackInterface.h

```diff
@@ -19,225 +19,11 @@
 #define NANOSTACK_INTERFACE_H_
```

Comment:
Why this header 'NanostackInterface.h' exists. It is just including other headers, a part which can be moved to its appropriate user class.

Reply:
Backwards compatibility - various apps include it.